### PR TITLE
Integrated renderer into the companion app service

### PR DESCRIPTION
**Pull request by erikos**
https://phabricator.endlessm.com/T21272

### CODE REVIEW

File: com.endlessm.CompanionAppService.json.in [hunk 1]
```diff
@@ -24,6 +24,27 @@
     ],
     "modules": [
         "com.endlessm.CompanionAppService.PipDependencies.json",
+        {
+            "name": "mustache-c",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/x86-64/mustache-c"
```
Comment (smspillaz) [inline com.endlessm.CompanionAppService.json.in]: Ah, I forgot to mention that you'll need to pin this to a certain version. See https://github.com/endlessm/endless-sdk-flatpak/blob/master/com.endlessm.apps.Sdk.json.in.in#L511

File: com.endlessm.CompanionAppService.json.in [hunk 2]
```diff
@@ -24,6 +24,27 @@
...
+                }
+            ]
+        },
+        {
+            "name": "meos-knowledge-content-renderer",
```
Comment (smspillaz) [inline com.endlessm.CompanionAppService.json.in]: `meos`

File: com.endlessm.CompanionAppService.json.in [hunk 3]
```diff
@@ -24,6 +24,27 @@
...
+            "buildsystem": "meson",
+            "builddir": true,
```
Comment (smspillaz) [inline com.endlessm.CompanionAppService.json.in]: I don't think `builddir` is necessary when using `meson` - it is enforced.

File: com.endlessm.CompanionAppService.json.in [hunk 4]
```diff
@@ -24,6 +24,27 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/endlessm/eos-knowledge-content-renderer",
+                    "branch": "T21272"
```
Comment (smspillaz) [inline com.endlessm.CompanionAppService.json.in]: Quick note here that we'll need to remove this branch once eknr is merged.

File: com.endlessm.CompanionAppService.json.in (hunk 1 above)
Comment (erikos) [reply com.endlessm.CompanionAppService.json.in]: Done.

File: com.endlessm.CompanionAppService.json.in (hunk 2 above)
Comment (erikos) [reply com.endlessm.CompanionAppService.json.in]: Oups, done.

File: com.endlessm.CompanionAppService.json.in (hunk 3 above)
Comment (erikos) [reply com.endlessm.CompanionAppService.json.in]: Done.

File: com.endlessm.CompanionAppService.json.in (hunk 4 above)
Comment (erikos) [reply com.endlessm.CompanionAppService.json.in]: Indeed.

**Comment by smspillaz** [comment]
Superceded by https://github.com/endlessm/eos-companion-app-integration/pull/83

**Pull request closed without merging**